openstack/path_static_role.go

```diff
@@ -138,11 +128,12 @@ func (b *backend) staticRoleExistenceCheck(ctx context.Context, r *logical.Reque
 type roleStaticEntry struct {
 	Name             string            `json:"name"`
 	Cloud            string            `json:"cloud"`
-	Root             bool              `json:"root"`
 	TTL              time.Duration     `json:"ttl,omitempty"`
 	RotationDuration time.Duration     `json:"rotation_duration,omitempty"`
 	SecretType       secretType        `json:"secret_type"`
+	Secret           string            `json:"secret"`
 	Username         string            `json:"username"`
+	UserId           string            `json:"user_id"`
 	ProjectID        string            `json:"project_id"`
 	ProjectName      string            `json:"project_name"`
 	DomainID         string            `json:"domain_id"`
```

Comment:
It wasn't done before, but it's a good idea to document structures, even private, especially that big

Reply:
Can you please elaborate where should I document it? Should I add all entries inside a schema?

Reply:
Yes, just a doc comment inside the structure

Reply:
user_id was missing in schema, added in latest commit

Reply:
For user ID: other IDs are `ID`, and this one is `Id`

Reply:
updated

Reply:
The original comment was not resolved, but it's okay